src/common/float_type.h

```diff
@@ -0,0 +1,43 @@
+#ifndef __FLOAT_TYPE_H_
+#define __FLOAT_TYPE_H_
+
+#include <stdint.h>
+
```

Comment:
I group together all vector types because I need them in a lot of file like specific file fro `vprec` (`vprec_tools` and `float_struct`) and I think it is not good to include the file `interflop.h` just to have these vector type, also 2 separated files for usual and specialized types.